src/melee/ft/chara/ftMars/ftMarsSpecialN.c

```diff
@@ -318,7 +318,7 @@ void lbl_80137010(HSD_GObj *gobj) {
         //         Fighter_BonePersonalToCommon
         func_8000B1CC(fp->x5E8_fighterBones[func_8007500C(fp, 4)].x0_jobj, 0, &position);
         // AestheticWindEffect(r3=sourcelocation,r4=duration,f1=radiusSize,f2=effectdegradation,f3=unk)
-        func_800119DC(&position, 120, 0.9f, 0.02f, 1.0471975803375244f);
+        func_800119DC(&position, 120, 0.9f, 0.02f, 60 * M_PI / 180);
     }
     // FrameTimerCheck
     if (ftAnim_IsFramesRemaining(gobj) == 0) {
```

Comment:
This is a `BOOL` not a frame count, so it's safe to replace it with 
```suggestion
    if (!ftAnim_IsFramesRemaining(gobj)) {
```

Appears quite a lot.

Reply:
Oof there are a ton of these. Think I took care of all of them.